### PR TITLE
Update the UG and finalise formatting

### PR DESCRIPTION
**Pull request by appleraincoat**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -334,18 +364,18 @@ Should you want to re-enter your contacts in a fresh JSON file in the event of f
 simply delete `realodex.json` and restart the app. A new JSON file with sample contacts will be generated and you may proceed from there.
 
 --------------------------------------------------------------------------------------------------------------------
-## Format Requirements
+## Field Constraints
```
Comment (4llysa) [inline docs/UserGuide.md:367]: If you rename this to Field Constraints, need to rename format-requirements to field-constraints in default.md also

File: docs/UserGuide.md [hunk 2]
```diff
@@ -75,81 +73,85 @@ command to run the application.<br>
...
+- When editing the `TAG`, all existing tags will be overwritten with the new tag(s) provided. If you want to edit the client to be both a buyer and seller, include both tags i.e. `t/Buyer t/Seller`.
+- All fields must follow the respective [Field Constraints](#field-constraints).
 
-Examples:
+<u>Examples</u>:
 
-- `edit 1 p/999` will overwrite the 1st client's phone number to `999`.
-- `edit 2 n/Kylie  i/3333 f/5` will overwrite the 2nd client's name to `Kylie`, income to `3333` and family size to `5`.
+- `edit 1 p/999` will overwrite the 1st client's phone number to "999".
+- `edit 2 n/Kylie  i/3333 f/5` will overwrite the 2nd client's name to "Kylie", income to "3333" and family size to "5".
```
Comment (UdhayaShan1) [inline docs/UserGuide.md:154]: Good!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -213,11 +224,12 @@ tag(s) specified in the input.
 - Supports searching with **multiple tags**.
   - `filter t/Buyer t/Seller` only returns persons with both "Buyer" and "Seller" tags.
 - **Comprehensive searching**, returning all persons' with the specified tag(s).
-    - `filter t/Seller` returns all persons tagged as "Seller".
```
Comment (jyztintan) [inline docs/UserGuide.md:216]: why delete uh

**Review by jyztintan — APPROVED** [review]
thanks for standardising

File: docs/UserGuide.md (hunk 3 above)
Comment (appleraincoat) [reply docs/UserGuide.md:216]: The scrshot description has exactly the same message

**Review by UdhayaShan1 — APPROVED** [review]
LGTM, thanks for scanning

**Pull request merged**